content/posts/numpy/fellowship-program/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "NumPy's first Developer in Residence: Sayed Adel"
+date: 2022-11-30
+draft: false
+description: |
+  Introducing the NumPy Developer in Residence program, and its
```

Comment:
We went with NumPy Fellowship Program in the end, as in the first sentence of the main text. Let's use that in this description.